src/main/java/entities/Car.java

```diff
@@ -0,0 +1,158 @@
+package entities;
+
+import java.io.Serializable;
+import java.util.Objects;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+import javax.persistence.NamedQuery;
+
+
+@Entity
+@NamedQuery(name = "RenameMe.deleteAllRows", query = "DELETE from RenameMe")
```

Comment:
renameMe

Reply:
Oh geez, everytime.. I always forget that